README.md

```diff
@@ -48,7 +48,7 @@ DATABASES = {
 $ ./manage.py migrate
 $ ./manage.py createsuperuser
 
-$ ./manage.py ingest GMLFILE
+$ ./manage.py ingest GMLFILE 160229_130000
```

Comment:
I disagree - humans can read blocks of characters much faster then a long string of characters. My format is not perfect since five characters per group would be optimal to recognize.